### ISSUE TITLE
error[internal]: left behind trailing whitespace:   "let x =  \n <very long comment>\n<very long code>"

### ISSUE THREAD

**Issue by matthiaskrgr** [rust-lang/rustfmt#4192]
````rust
fn main() {
    let x = 
    // wordwordwordwordwordwordwordwordwordwordwordwordwordwordwordwordwordwordwordword
    format!("This is a very long line with a lot of text it goes so very long, it's the best line!");
}
````
The space after `=` in `    let x =<space>` causes the internal error.

```
error[internal]: left behind trailing whitespace
 --> /tmp/fmt/src/main.rs:2:2:11
  |
2 |     let x =
  |            ^
  |

warning: rustfmt has failed to format. See previous 1 errors.
````

`rustfmt 1.4.14-nightly (a5cb5d2 2020-04-14)`

**Comment by calebcartwright** [comment]
This is fixed on master/rustfmt 2.0

**Comment by calebcartwright** [comment]
Going to close as there's nothing else we can do on our end, and the fix will hit nightly with rustfmt 2.0
